src/utils/uiHelper/MapHelper.js

```diff
@@ -14,3 +14,14 @@ export const getMyPosition = (success, error) => {
     maximumAge: 0
   });
 };
+
+export const mapMarkerToTarget = (marker, targetList) => {
```

Comment:
You can get only the props `latitude` and `longitude` of the marker:

`export const mapMarkerToTarget = ({ latitude, longitude }, targetList) => {`

Then you can write the if condition inline:
`if ((target.lat === latitude) && (target.lng === longitude)) {`